integrations/repo_commits_search_test.go

```diff
@@ -28,7 +28,10 @@ func testRepoCommitsSearch(t *testing.T, query, commit string) {
 }
 
 func TestRepoCommitsSearch(t *testing.T) {
+	testRepoCommitsSearch(t, "58e97", "58e97d1a24")
+	testRepoCommitsSearch(t, "6e8e", "6e8eabd9a7")
```

Comment:
Could we have a failed test.

Reply:
Sorry, I'm not sure what you mean by that.

Reply:
I think to have a search that should *not* match anything, to test that it works right when there is nothing to match also.

So if there is no commit ```58f73d1a78``` then do a search for that and make sure it returns empty and not something else.

Reply:
Okay, sounds good.

Reply:
Alright, added a few.